docs/migrate-from-sidecar-to-ambient/migration.adoc

```diff
@@ -0,0 +1,1604 @@
+3. **Pod labeling**: Adding the label to individual pods (requires `istio.io/waypoint-for: workload`)
+
+**Traffic Flow Decision Tree:**
+- **Without waypoint labels**: Traffic flows directly through ztunnel (L4-only processing)
+- **With waypoint labels**: Traffic is intercepted and routed through waypoint proxies for L7 processing
+- **Mixed environments**: Some services use waypoints while others remain ztunnel-only
+
+**Decision Criteria:**
+
+- **Use waypoints if**: You need VirtualServices, HTTPRoutes, L7 AuthorizationPolicies, fault injection, or HTTP-level observability
+- **Skip waypoints if**: You only need mTLS, L4 AuthorizationPolicies, and basic connectivity
+- **Hybrid approach**: Deploy waypoints only for services requiring L7 features
+
+**Important**: Waypoint activation is explicit, deploying a waypoint Gateway resource alone does not affect traffic flow. You must label namespaces, services, or pods to direct traffic through waypoints.
+
+For detailed waypoint configuration guidance, see link:../common/istio-ambient-waypoint.adoc[Waypoint Proxy Documentation].
+
+[[compatibility-validation]]
+=== 2.2 Compatibility between Sidecar and Ambient Modes
+
+**Supported Features:**
+
+- All L4 traffic policies (AuthorizationPolicy, PeerAuthentication)
+- L7 traffic management (VirtualService, DestinationRule converted to HTTPRoute)
+- Security policies (RequestAuthentication, AuthorizationPolicy)
+- Telemetry and observability features
+
+**Unsupported Features** (migration blockers):
+
+- Multi-cluster mesh configurations
+- Virtual Machine (VM) workload integration
+- SPIRE integration for identity federation
+
+For additional compatibility considerations refer to:
+- link:../common/coexistence.md [Sidecar-Ambient Coexistence Guidelines]
+
+[[pre-migration-checklist]]
+=== 2.3 Pre-Migration Checklist
+
+**Environment Validation:**
+[source,console]
+----
+# Verify Sail Operator is deployed
+kubectl get pods -n sail-operator
+----
+
+[source,console]
+----
+# Check Istio control plane version
+kubectl get istio -n istio-system
+NAME      NAMESPACE      PROFILE   REVISIONS   READY   IN USE   ACTIVE REVISION   STATUS    VERSION   AGE
+default   istio-system             1           1       1        default           Healthy   v1.27.0   20m
+----
+
+[source,console]
+----
+# Verify no unsupported features are in use
+kubectl get virtualservice,destinationrule -A
+----
+
+Note: if you have any of the unsupported features listed above, we will need to resolve those before proceeding with the migration. Take into account that ambient support PeerAuthentication but disable mode it's not supported, more information about this can be found here: https://istio.io/latest/docs/reference/config/security/peer_authentication/.
+
+**Application Namespace Validation:**
+
+[source,console]
+----
+# Check your application namespaces for sidecar injection
+kubectl get namespaces -l istio.io/rev=default
```

Comment:
When I have istio cr with `default` name, I can have only `istio-injection=enabled` label on application (bookinfo) project.
in that case, I will not see any project here. 

Can you please extend about
```
kubectl get namespaces -l istio-injection=enabled
```